e2e-tests/web/test/pageobjects/page.ts

```diff
@@ -9,7 +9,11 @@ export default class Page {
    * @param path path of the sub page (e.g. /path/to/page.html)
    */
 
-  open(path: string): Promise<string> {
-    return browser.url(`${path}`)
+  async open(path: string): Promise<string> {
```

Comment:
:upside_down_face: nitpick: i personally dislike mixing async/await and Promises like this, but no need to change it.

Reply:
How would you change this? I think I don't really mix async/await and promises, `Promise` is just the correct return type of a async function.